### PR TITLE
Add --no-advice to perf report, small fixes

### PR DESCRIPTION
**Pull request by yieldthought**
### Ticket
n/a

### Problem description
Small improvements to the perf report script

### What's changed
- Add `--no-advice` to skip the advice section
- Do not use the , separator in the ID column
- Improve the fidelity advice
- Use us instead of µs for compatibility with more terminals

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11012767518)
- [ ] Blackhole Post commit (n/a)
- [ ] Model regression CI testing passes (n/a)
- [ ] Device performance regression CI testing passes (n/a)
- [ ] New/Existing tests provide coverage for changes (n/a)

### CODE REVIEW

**Comment by yieldthought** [comment]
Making myself the CODEOWNER for this script, it doesn't affect anything else in CI and it'd be a lot better if I could push changes to it without requiring review from another timezone.

**Review by tt-rkim — APPROVED** [review]

**Pull request merged**
